test/intl402/PluralRules/prototype/constructor/notation.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-initializepluralrules
+description: Checks that the notation option is picked up correctly.
+info: |
+  Intl.PluralRules ( [ _locales_ [ , _options_ ] ] )
+    ...
+    1. Let notation be ? GetOption(options, "notation", "string", « "standard", "compact", "scientific", "engineering" », "standard").
+    ...
+---*/
+
+const validValues = ["standard", "compact", "scientific", "engineering", new String("standard"), new String("compact"), new String("scientific"), new String("engineering")];
+const invalidValues = ["COMPACT", "ståndard", 123, false, Symbol("foo"), null, {}, [], ""];
+
+for (const value of validValues) {
+  assert.doesNotThrow(() => {
```

Comment:
`assert.doesNotThrow` is not a thing :smile: Either just `new Intl.PluralRules()` and do nothing with the created instance, or assert that it is of the correct type.